wgsl/index.bs

```diff
@@ -10076,10 +10076,10 @@ The rules for analyzing expressions take as argument both the expression itself
       where the identifier appears as the [=root identifier=] of a [=memory view=]
       expression, *MVE*, and the [=load rule=] is invoked on *MVE* during
       [=type checking=]
-      <td>*Result*
+      <td>[=Result=]
```

Comment:
Ooops. This is a newly introduced node. It should not refer to the definition of Result in the function call section....